api/src/test/java/org/openmrs/api/impl/ConditionServiceImplTest.java

```diff
@@ -30,7 +29,7 @@
  */
 public class ConditionServiceImplTest extends BaseContextSensitiveTest {
 	
-	protected static final String CONDITION_XML = "org/openmrs/api/include/ConditionServiceImplTest-SetupCondition.xml";
+	protected static final String DIAGNOSIS_XML = "org/openmrs/api/include/ConditionServiceImplTest-SetupCondition.xml";
```

Comment:
@pluwum Can you revert changes in this file?

Reply:
@paradisekelechi sorry about that :)